FindADIOS.cmake

```diff
@@ -28,42 +29,92 @@
 # This module will define the following variables:
 #   ADIOS_INCLUDE_DIRS    - Include directories for the ADIOS headers.
 #   ADIOS_LIBRARIES       - ADIOS libraries.
+#   ADIOS_DEFINITIONS     - ADIOS compile definitions.
 #   ADIOS_FOUND           - TRUE if FindADIOS found a working install
 #   ADIOS_VERSION         - Version in format Major.Minor.Patch
+#   ADIOS_HAVE_SEQUENTIAL - TRUE if found library links as sequential only
 #
 # Not used for now:
 #   ADIOS_DEFINITIONS     - Compiler definitions you should add with
 #                           add_definitions(${ADIOS_DEFINITIONS})
 #
+# Example to find ADIOS (default)
+# find_package(ADIOS)
+# if(ADIOS_FOUND)
+#   include_directories(${ADIOS_INCLUDE_DIRS})
+#   add_executable(foo foo.c)
+#   target_link_libraries(foo ${ADIOS_LIBRARIES})
+# endif()
+
+# Example to find ADIOS using component
+# find_package(ADIOS COMPONENTS fortran)
+# if(ADIOS_FOUND)
+#   include_directories(${ADIOS_INCLUDE_DIRS})
+#   add_executable(foo foo.c)
+#   target_link_libraries(foo ${ADIOS_LIBRARIES})
+# endif()
+###############################################################################
+#Copyright (c) 2014, Axel Huebl and Felix Schmitt from http://picongpu.hzdr.de
+#All rights reserved.
+
+#Redistribution and use in source and binary forms, with or without
```

Comment:
@ax3l do you changed by accident the license here? 

CC-ing: @sbastrakov

Reply:
I missed that in my review

Reply:
Good question, @c0nsultant checked in my script to openPMD-api in 2017 with this extended notice
https://github.com/openPMD/openPMD-api/commit/83edf0b4e6347761afad3f330954299f7ecca713
But this is still 3-clause/New BSD License license, so good to go. Paragraphs were actually missing in the original text.

Reply:
Okay, just so that there is no confusion, I meant not just the header change, but that the years moved back from 2014-2015 to just 2014.